docs/integrations/sources/mssql.md

```diff
@@ -327,6 +327,7 @@ WHERE actor_definition_id ='b5ea17b1-f170-46dc-bc31-cc744ca984c1' AND (configura
 
 | Version | Date       | Pull Request                                                                                                          | Subject                                                                                                                                         |
 |:--------|:-----------|:----------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------|
+| 4.0.5   | 2024-03-20 | [36340](https://github.com/airbytehq/airbyte/pull/36340)                                                              | [Refactor] : Consolidate extra metadata check.                                                                                                  |
```

Comment:
Words like "refactor" should never make it into a changelog. The audience for the changelog is the user, who doesn't care about internal changes.